toolkit/pythonit_toolkit/api/service_client.py

```
@@ -0,0 +1,47 @@
+from dataclasses import dataclass
+from typing import Any, Dict, Optional
+
+import httpx
+from pythonit_toolkit.headers import SERVICE_JWT_HEADER
+from pythonit_toolkit.pastaporto.tokens import generate_token
+
+
+@dataclass
+class ServiceResponse:
+    errors: Optional[Dict[str, Any]]
+    data: Optional[Dict[str, Any]]
+
+
+class ServiceClient:
+    def __init__(
+        self,
+        url: str,
+        issuer: str,
+        audience: str,
+        jwt_secret: str,
+    ):
+        self.url = url
+        self.issuer = issuer
+        self.audience = audience
+        self.jwt_secret = jwt_secret
```

we should raise an error if those are empty